.ai/sail/core.blade.php

```diff
@@ -0,0 +1,14 @@
+@php
+/** @var \Laravel\Boost\Install\GuidelineAssist $assist */
+@endphp
+## Laravel Sail
+
+- This project runs inside Laravel Sail's Docker containers. You MUST execute all commands through Sail.
+- Start services using `{{ $assist->sailBinaryPath() }} up -d` and stop them with `{{ $assist->sailBinaryPath() }} stop`.
+- Open the application in the browser by running `{{ $assist->sailBinaryPath() }} open`.
+- Always prefix PHP, Artisan, Composer, and Node commands** with `{{ $assist->sailBinaryPath() }}`. Examples:
```

Comment:
Hey @pushpak1300 I was just wondering, did you meant to connect up these two asterisks after "commands" to emphasize something?  They stood out to me, looking a bit out of place all by themselves.